paddle/fluid/platform/device_context.cc

```diff
@@ -441,10 +456,17 @@ void MKLDNNDeviceContext::SetBlob(const std::string& name,
   }
 
   // Find KeyBlob for current input shape
-  std::string cur_input_shape_str = platform::get_cur_input_shape_str();
   auto key_it = sBlob->find(cur_input_shape_str);
 
   if (key_it == sBlob->end()) {
+    // In cache clearing mode, cur_input_shape_cache_size defines max pblob
+    // capacity
+    if ((tid == kMKLDNNSessionID_CacheClearing) &&
```

Comment:
better rename tid to sid (session id) to align with kMKLDNNSessionID_xxx

Reply:
Got it. I will change.